server/src/main/java/com/microsoft/java/bs/core/Launcher.java

```diff
@@ -36,12 +40,40 @@ public class Launcher {
   public static void main(String[] args) {
     checkRequiredProperties();
 
-    org.eclipse.lsp4j.jsonrpc.Launcher<BuildClient> launcher = createLauncher();
+    String pipePath = null;
+    for (String arg : args) {
+      if (arg.startsWith("--pipe=")) {
```

Comment:
Usually, the best practice of design the application argument is --<option name> --<option value>

<foo=bar> is more like system property.